python/src/ecole/core/observation.cpp

```diff
@@ -111,6 +112,37 @@ void bind_submodule(py::module m) {
 	def_reset(
 		node_bipartite, "Cache some feature not expected to change during an episode.");
 	def_obtain_observation(node_bipartite, "Extract a new :py:class:`NodeBipartiteObs`.");
+
+	py::class_<StrongBranchingScoresObs>(m, "StrongBranchingScoresObs", R"(
+		Strong Branching Scores for each variable at a branch-and-bound node.
+
+		This observation obtains scores for all LP or pseudo candidate variables at a 
+		branch-and-bound node.  The strong branching score measures the quality of branching 
+		for each variable.  This observation can be used as an expert in reinforcement
+		learning algorithms. 
+	)")  //
+		.def_property_readonly(
+			"scores",
+			[](StrongBranchingScoresObs & self) -> auto& { return self.scores; },
+			"An array containing the strong branching score for each variable in the problem "
+			"which can be indexed by the action set.  Variables for which a strong branching "
+			"score is not applicable are filled with NaN.");
+
+	auto strong_branching_scores =
+		py::class_<StrongBranchingScores>(m, "StrongBranchingScores", R"(
+		Strong branching score observation function on branch-and bound node.  The parameter
+		pseudo_candidates can be passed as a boolean to compute strong branching scores for 
+		pseudo-candidate variables if true and LP candidates variables if false.  By default
+		psuedo-candidates will be computed.  
+
+		This observation function extract structured :py:class:`StrongBranchingScoresObs`.
```

Comment:
or long docstring the convention is similar to what you did with `StrongBranchingScoresObs`
```python
"""One sentence short summary.

Longer description.
"""
```
https://numpydoc.readthedocs.io/en/latest/format.html#sections